server/src/test/java/org/opensearch/search/fetch/subphase/highlight/HighlightFieldTests.java

```diff
@@ -53,19 +53,32 @@ public class HighlightFieldTests extends OpenSearchTestCase {
 
     public static HighlightField createTestItem() {
         String name = frequently() ? randomAlphaOfLengthBetween(5, 20) : randomRealisticUnicodeOfCodepointLengthBetween(5, 20);
```

Comment:
Any reason why we didn't modify `randomAlphaOfLengthBetween` and `randomRealisticUnicodeOfCodepointLengthBetween` to not return unicode control characters?